cli/flox/src/utils/errors.rs

```diff
@@ -397,7 +397,7 @@ pub fn format_managed_error(err: &ManagedEnvironmentError) -> String {
         .to_string(),
 
         ManagedEnvironmentError::ReadLocalManifest(_) => display_chain(err),
-        ManagedEnvironmentError::ReadGenerationManifest(_) => display_chain(err),
+        ManagedEnvironmentError::Generations(_) => display_chain(err),
```

Comment:
(unrelated line)
suggestion nonblocking: should we add a regression test to ensure we don't clone for an `activate` command?

Reply:
If you still think we should add a test, a simple, stupid one would be 

```bash
run flox activate -v -- true
refute_output "clone"
```

but i feel thats a bit wonky, depending on log output like that..